gradoop-temporal/src/test/java/org/gradoop/temporal/util/TimeFormatConversionTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.temporal.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.time.LocalDateTime;
+
+import static org.gradoop.temporal.util.TimeFormatConversion.convertLocalDateTimeToEpochMilli;
+
+public class TimeFormatConversionTest {
+
+  @Test
+  public void convertLocalDateTimeToEpochMilliTest() {
+    LocalDateTime dateTime = LocalDateTime.of(1970, 1, 1, 0, 0);
+    Assert.assertEquals(0, convertLocalDateTimeToEpochMilli(dateTime));
```

Comment:
Please add one more example. If there is a bug that always returns 0L we will not be noticed.